ni_measurementlink_service/_configuration.py

```diff
@@ -0,0 +1,108 @@
+"""MeasurementLink configuration options."""
+from __future__ import annotations
+
+import pathlib
+import sys
+from typing import TYPE_CHECKING, Any, Callable, Dict, NamedTuple, TypeVar, Union
+
+from decouple import AutoConfig, Undefined, undefined
+
+if TYPE_CHECKING:
+    if sys.version_info >= (3, 11):
+        from typing import Self
+    else:
+        from typing_extensions import Self
+
+
+_PREFIX = "MEASUREMENTLINK"
+
+# Search for the `.env` file starting with the current directory.
+_config = AutoConfig(str(pathlib.Path.cwd()))
+
+if TYPE_CHECKING:
+    # Work around decouple's lack of type hints.
+    _T = TypeVar("_T")
+
+    def _config(
+        option: str,
+        default: Union[_T, Undefined] = undefined,
+        cast: Union[Callable[[str], _T], Undefined] = undefined,
+    ) -> _T:
+        ...
```

Comment:
Is this some kind of notation for 'pass'?

Reply:
It's effectively equivalent (in this context):
https://docs.python.org/3/library/constants.html#Ellipsis

Community hasn't decided on a convention for when to use which for empty contexts.
https://stackoverflow.com/a/55275342

Reply:
This is a type stub and type stubs use ellipses.